.github/workflows/Tests.yaml

```diff
@@ -438,3 +438,59 @@ jobs:
           # We get occasional random timeouts, repeat tests to see if
           # it is a random timeout or systematic
           ctest -j2 --output-on-failure -L inputfiles --repeat after-timeout:3
+      - name: Test formaline tar can be built
+        working-directory: /work/build
+        run: >
+          make EvolveBurgersStep -j2
+
+          if [ ! -f ./bin/EvolveBurgersStep ]; then
+            echo "Could not find the executable EvolveBurgersStep";
+            echo "which we use for testing formaline";
+            exit 1
+          fi
+
+          ASAN_OPTIONS=detect_leaks=0 ./bin/EvolveBurgersStep
+          --dump-source-tree-as spectre_src --dump-only
+
+          mkdir spectre_src;
+          mv spectre_src.tar.gz spectre_src;
+          cd spectre_src;
+          tar xf spectre_src.tar.gz;
+          mkdir build;
+          cd build
+
+          if [[ ${{ matrix.compiler }} =~ (gcc|clang)-([0-9\.]+) ]]; then
+            CC=${BASH_REMATCH[1]}-${BASH_REMATCH[2]};
+            CHARM_CC=${BASH_REMATCH[1]};
+            if [[ ${BASH_REMATCH[1]} = gcc ]]; then
+              CXX=g++-${BASH_REMATCH[2]};
+              FC=gfortran-${BASH_REMATCH[2]};
+            else
+              CXX=clang++-${BASH_REMATCH[2]};
+              FC=gfortran-8;
+            fi
+          fi
+
+          MEMORY_ALLOCATOR=${{ matrix.MEMORY_ALLOCATOR }};
+          USE_PCH=${{ matrix.use_pch }};
+
+          cmake
+          -D CMAKE_C_COMPILER=${CC}
+          -D CMAKE_CXX_COMPILER=${CXX}
+          -D CMAKE_Fortran_COMPILER=${FC}
+          -D CMAKE_CXX_FLAGS="${CXXFLAGS} ${{ matrix.EXTRA_CXX_FLAGS }}"
+          -D OVERRIDE_ARCH=x86-64
+          -D CHARM_ROOT=/work/charm_6_10_2/multicore-linux-x86_64-${CHARM_CC}
+          -D CMAKE_BUILD_TYPE=${{ matrix.build_type }}
+          -D DEBUG_SYMBOLS=OFF
+          -D RUN_TESTS_IN_TEST_EXECUTABLES=OFF
+          -D STRIP_SYMBOLS=ON
+          -D STUB_EXECUTABLE_OBJECT_FILES=ON
+          -D STUB_LIBRARY_OBJECT_FILES=ON
+          -D USE_PCH=${USE_PCH}
+          -D USE_CCACHE=OFF
+          -D BUILD_PYTHON_BINDINGS=OFF
+          -D MEMORY_ALLOCATOR=${MEMORY_ALLOCATOR:-'SYSTEM'}
+          ..
+
+          make EvolveBurgersStep -j2
```

Comment:
also run `ctest -R InputFiles.Burgers...` just to round off the test? Please squash directly if you add this